contracts/solidity/defaults/Defaults.sol

```diff
@@ -0,0 +1,108 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.20;
+
+contract Defaults {
+    struct UintDefaults {
+        uint uInt;
+        uint8 uInt8;
+        uint16 uInt16;
+        uint32 uInt32;
+        uint64 uInt64;
+        uint128 uInt128;
+        uint256 uInt256;
+    }
+
+    struct IntDefaults {
+        int intDef;
+        int8 intDef8;
+        int16 intDef16;
+        int32 intDef32;
+        int64 intDef64;
+        int128 intDef128;
+        int256 intDef256;
+    }
+
+    // struct FixedDefaults {
+    //     fixed fixedVar;
+    //     fixed8x18 fixed8x18Var;
+    //     fixed16x12 fixed16x12Var;
+    //     fixed32x10 fixed32x10Var;
+    //     fixed64x8 fixed64x8Var;
+    //     fixed128x6 fixed128x6Var;
+    //     fixed256x4 fixed256x4Var;
+    // }
+
+    // struct UFixedDefaults {
+    //     ufixed ufixedVar;
+    //     ufixed8x18 ufixed8x18Var;
+    //     ufixed16x12 ufixed16x12Var;
+    //     ufixed32x10 ufixed32x10Var;
+    //     ufixed64x8 ufixed64x8Var;
+    //     ufixed128x6 ufixed128x6Var;
+    //     ufixed256x4 ufixed256x4Var;
+    // }
+
+    struct BytesDefaults {
+        bytes3 bytesDef3;
+        bytes10 bytesDef10;
+        bytes15 bytesDef15;
+        bytes20 bytesDef20;
+        bytes25 bytesDef25;
+        bytes30 bytesDef30;
+        bytes32 bytesDef32;
+    }
+
+    struct ArrayDefaults {
+        string[] strArr;
+        uint[] uintArr;
+        bool[] boolArr;
+        bytes[] bytesArr;
+    }
+
+    mapping(string => uint) public strUintMap;
+    mapping(address => bool) public addrBoolMap;
+    mapping(int => bytes) public bytesBytesMap;
+
+    function getUintDefaults() external pure returns (UintDefaults memory) {
+        UintDefaults memory defaults;
+        return defaults;
+    }
+
+    function getIntDefaults() external pure returns (UintDefaults memory) {
+        UintDefaults memory defaults;
+        return defaults;
+    }
+
+    // Not supported by solidity yet
```

Comment:
Q: Interesting, any docs to add a note here?

Reply:
You can find the documentation: [here](https://docs.soliditylang.org/en/latest/types.html#fixed-point-numbers)

<img width="606" alt="image" src="https://github.com/hashgraph/hedera-smart-contracts/assets/10372129/2b6f0e54-41c1-4a76-8cf4-822a8b34c955">

Reply:
Yes, I meant you could add an in code comment so that it's not perceived as Hedera doesn't support it and rather the language itself.

Reply:
Done